packages/color-area/src/ColorArea.ts

```diff
@@ -0,0 +1,389 @@
+import '@spectrum-web-components/color-handle/sp-color-handle.js';
+import { TinyColor } from '@ctrl/tinycolor';
+
+import styles from './color-area.css.js';
+
+/**
+ * @element sp-color-area
+ */
+export class ColorArea extends SpectrumElement {
+    public static get styles(): CSSResultArray {
+        return [styles];
+    }
+
+    @property({ type: Boolean, reflect: true })
+    public disabled = false;
+
+    @property({ type: Boolean, reflect: true })
+    public focused = false;
+
+    @query('.handle')
+    private handle!: ColorHandle;
+
+    @property({ type: Number })
+    public get hue(): number {
+        return this._hue;
+    }
+
+    public set hue(value: number) {
+        const hue = Math.min(360, Math.max(0, value));
+        if (hue === this.hue) {
+            return;
+        }
+        const oldValue = this.hue;
+        const { s, v } = this._color.toHsv();
+        this._color = new TinyColor({ h: hue, s, v });
+        this._hue = hue;
+        this.requestUpdate('hue', oldValue);
+    }
+
+    private _hue = 0;
+
+    @property({ type: String })
+    public get value(): ColorValue {
+        return this._value.toHex();
+    }
+
+    private _value = new TinyColor({ h: 0, s: 1, v: 1 });
+
+    @property({ type: String })
+    public get color(): ColorValue {
+        switch (this._format[0]) {
+            case 'rgb':
+                return this._format[1]
+                    ? this._color.toRgbString()
+                    : this._color.toRgb();
+            case 'prgb':
+                return this._format[1]
+                    ? this._color.toPercentageRgbString()
+                    : this._color.toPercentageRgb();
+            case 'hex8':
+                return this._format[1]
+                    ? this._color.toHex8String()
+                    : this._color.toHex8();
+            case 'name':
+                return this._color.toName() || this._color.toRgbString();
+            case 'hsl':
+                return this._format[1]
+                    ? this._color.toHslString()
+                    : this._color.toHsl();
+            case 'hsv':
+                return this._format[1]
+                    ? this._color.toHsvString()
+                    : this._color.toHsv();
+            case 'hex':
+            case 'hex3':
+            case 'hex4':
+            case 'hex6':
+            default:
+                return this._format[1]
+                    ? this._color.toHexString()
+                    : this._color.toHex();
+        }
+    }
+
+    public set color(color: ColorValue) {
+        if (color === this.color) {
+            return;
+        }
+        const oldValue = this._color;
+        this._color = new TinyColor(color);
+        this._format = [
+            this._color.format,
+            typeof color === 'string' || color instanceof String,
+        ];
+        const { h, s, v } = this._color.toHsv();
```

Comment:
I don't quite understand what you're asking. The benefit of having `hue` exposed here, though, is that you can manually adjust the slider (so, the `hue` value specifically) in the console separately from the colour area. If you wanted to adjust the entire colour settings (ie inputting a hex value instead of only adjusting the hue), you have the freedom to do that.